SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Reporting Security Vulnerabilities
+
+BrowserStack Community take all security vulnerabilities seriously. Thank you for improving the security of our open source software. We appreciate your efforts and responsible disclosure and will make every effort to acknowledge your contributions.
```

Comment:
`We appreciate your efforts and responsible disclosure and will make every effort to acknowledge your contributions.` - Please update this line to avoid multiple use of `and`